src/main/java/org/jitsi/jicofo/FocusManager.java

```diff
@@ -284,44 +279,33 @@ public void start()
             this.jicofoId = jicofoId;
         }
 
-        focusUserDomain = JidCreate.domainBareFrom(
-                config.getString(FOCUS_USER_DOMAIN_PNAME));
-        focusUserName = Resourcepart.from(
-                config.getString(FOCUS_USER_NAME_PNAME));
-
-        String focusUserPassword = config.getString(FOCUS_USER_PASSWORD_PNAME);
-        String xmppServerPort = config.getString(XMPP_PORT_PNAME);
+        ClientConnectionConfig clientConnectionConfig = xmppConfig.getClientConnectionConfig();
 
         // We default to "conference" prefix for the muc component
-        String conferenceMucPrefix
-            = config.getString(XMPP_MUC_COMPONENT_PREFIX_PNAME, "conference");
-        conferenceMucService = JidCreate.domainBareFrom(
-            conferenceMucPrefix + "." + xmppDomainConfig);
+        String conferenceMucPrefix = config.getString(XMPP_MUC_COMPONENT_PREFIX_PNAME, "conference");
```

Comment:
Why is this property still in the old config?  Is it not part of `ClientConnectionConfig`?

Reply:
Because I split the changes in multiple PRs.